packages/eslint-config-small-ads/index.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "plugin:react-hooks/recommended",
+    "airbnb-typescript",
+    "prettier",
+    "prettier/react",
+    "prettier/@typescript-eslint",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: "module",
+  },
+  plugins: ["react-hooks", "@typescript-eslint/eslint-plugin"],
+  rules: {
+    "import/prefer-default-export": "off",
+    "import/no-extraneous-dependencies": "off",
```

Comment:
You dont want to deactivate this for _all_ our packages. You can do that extending the rules only for the ui package. Remember the princile [Open-Closed Principle](https://www.digitalocean.com/community/conceptual_articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design#open-closed-principle). Modules should be extendable without having to manipulate the module itself.

Reply:
I understand now, thanks for the explanation and the and for making me think about how to extend modules. I can see how it is better to have particular configuration for each project to their own `.eslintrc.json`.